pkg/templates/templates.go

```diff
@@ -661,6 +699,29 @@ func (t *TemplateResolver) ResolveTemplate(
 		funcMap[customFuncName] = customFunc
 	}
 
+	// Wrap any denylisted functions so that using them results in a clear error at
+	// template execution time instead of an undefined function error.
+	if len(options.DenylistFunctions) != 0 {
+		for _, funcName := range options.DenylistFunctions {
+			name := funcName
+
+			funcMap[name] = func(args ...interface{}) (interface{}, error) {
+				// Mark args as used to satisfy linters, even though we ignore them.
+				_ = args
```

Comment:
Can this just blank the `args` input or does that not work?

```suggestion
			funcMap[name] = func(_ ...interface{}) (interface{}, error) {
```